socorro/unittest/external/es/base.py

```diff
@@ -50,26 +50,13 @@ def test_with_version(self):
     return decorated
 
 
-class SuperSearchWithFields(SuperSearch):
-    """Convenience class for SuperSearch to pass all fields
-
-    SuperSearch's get method requires to be passed the list of all fields. This
-    class does that automatically so we can just use ``get()``.
-
-    """
-
-    def get(self, **kwargs):
-        kwargs["_fields"] = copy.deepcopy(FIELDS)
-        return super().get(**kwargs)
```

Comment:
Most of the tests that used this didn't really need it and I think it moves important information about the test too far from the test. So I moved this to `test_supersearch.py` where this is helpful since that file is testing Supersearch queries only.

For the rest of the files using this, I changed the `api.get()` calls to pass in `_fields`.